tests/testthat/helper.R

```diff
@@ -0,0 +1,21 @@
+skip_unless_authenticated <- function() {
```

Comment:
I guess there's no reason you can't encrypt an oauth token (and use in CI) in exactly the same way we do service account tokens. The only caveat is that they are more likely to grow stale or fall off the end. But that's really neither here nor there.

https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html

Reply:
Yeah I was going to try this eventually, for now I just wanted to get tests working for me locally with cached tokens + a few environment variables

Reply:
At least 2 other people have gotten it working from those instructions, so it shouldn't be too bad.